rulebook.tex

```diff
@@ -2025,34 +2079,48 @@ \section{Hardware Specification}
 \subsection{Robot Dimensions}
 \begin{figure}
   \includegraphics{figures/dimensions.pdf}
-  \caption{Robot dimensional constraints (top down view).}
+  \caption{Robot dimensional constraints (top down view), derived from the Robotino platform.}% chktex ignore-long-line
   \label{fig:dimensions}
 \end{figure}
 
+\subsection{Physical Constraints}
+The Festo Robotino\footnote{\href{https://robotino.com}{robotino.com}} by
```

Comment:
Given that the robotino is phasing out anyways, we could also get rid of platform-specific mentions of it alltogether.
The imposed limitations to speed and size dont need further justification IMO.
Just an Idea, I do not deeply care one way or another. It just feels unecessary to kinda highlight a platform that is not sold anymore.

Reply:
i mean we can just state an arbitrary limit like 0.5x0.5 m footprint (reasonable given zone size) and 1 m/s top speed (for safety reasons) and call it a day for now. If there is compelling reason to relax this in the future we can.
Faster robots can easily be restricted to be slower. For other (bigger) footprints we could add a TC approval clause if you fear of blocking potentially suitable platforms.